### PR TITLE
Add android and iOS media reoffer guide

### PR DESCRIPTION
**Pull request by abdulajet**
## Description

Adds a guide for the call reconnection feature
https://nexmo-developer-pr-3885.herokuapp.com/client-sdk/in-app-voice/guides/reconnect-call/

### CODE REVIEW

File: _documentation/en/client-sdk/in-app-voice/guides/reconnect-call.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+To know when the status of a call or conversation's media has changed you can listen for media status events with the Android and iOS SDK. Note the disconnected state. A disconnected state means that there has been a temporary network issue and the client will attempt a reconnection providing the `autoReoffer` configuration property is set. If you did not set `autoReoffer` to `true` then you can manually reconnect here. 
+
+```tabbed_content
+source: '_tutorials_tabbed_content/client-sdk/guides/reconnect-call/status'
+frameless: false
+```
+
+### Manually Reconnect
+
+The Client SDK has can also has functions for explicitly reconnecting a call or conversation's media. This is useful for example when you want to switch which device a user is speaking on without hanging up the call and starting a new one if the application dies.
```
Comment (horothesun) [inline _documentation/en/client-sdk/in-app-voice/guides/reconnect-call.md:33]:
```suggestion
The Client SDK has functions for explicitly reconnecting a call or conversation's media. This is useful for example when you want to switch which device a user is speaking on without hanging up the call and starting a new one if the application dies.
```

**Review by horothesun — APPROVED** [review]
LGTM! ✅ 👍

File: _documentation/en/client-sdk/in-app-voice/guides/reconnect-call.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+The Client SDK can attempt to automatically reconnect to a call or a conversation's media if you set the client's `autoReoffer` configuration property. This is useful for when there is a short drop in internet connectivity or the user switches between cellular and WiFi.
+
+```tabbed_content
+source: '_tutorials_tabbed_content/client-sdk/guides/reconnect-call/automatically-reconnect'
+frameless: false
+```
+
+### Listening for Call Status Changes
+
+To know when the status of a call or conversation's media has changed you can listen for media status events with the Android and iOS SDK. Note the disconnected state. A disconnected state means that there has been a temporary network issue and the client will attempt a reconnection providing the `autoReoffer` configuration property is set. If you did not set `autoReoffer` to `true` then you can manually reconnect here.
```
Comment (conshus) [inline _documentation/en/client-sdk/in-app-voice/guides/reconnect-call.md:24]:
```suggestion
To know when the status of a call or conversation's media has changed, you can listen for media status events with the Android and iOS SDK. Note the disconnected state. A disconnected state means that there has been a temporary network issue and the client will attempt a reconnection providing the `autoReoffer` configuration property was set. If you did not set `autoReoffer` to `true` then you can manually reconnect here. 
```

File: _tutorials_tabbed_content/client-sdk/guides/reconnect-call/status/java.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Java
+language: java
+menu_weight: 2
+---
+
+To get `MediaConnectionState` updates you need to add a `NexmoMediaStatusListener`. You can do this by setting it on a call's conversation object.
```
Comment (conshus) [inline _tutorials_tabbed_content/client-sdk/guides/reconnect-call/status/java.md:7]:
```suggestion
To get `MediaConnectionState` updates, you need to add a `NexmoMediaStatusListener`. You can do this by setting it on a call's conversation object.
```

File: _documentation/en/client-sdk/in-app-voice/guides/reconnect-call.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+To know when the status of a call or conversation's media has changed you can listen for media status events with the Android and iOS SDK. Note the disconnected state. A disconnected state means that there has been a temporary network issue and the client will attempt a reconnection providing the `autoReoffer` configuration property is set. If you did not set `autoReoffer` to `true` then you can manually reconnect here. 
+
+```tabbed_content
+source: '_tutorials_tabbed_content/client-sdk/guides/reconnect-call/status'
+frameless: false
+```
+
+### Manually Reconnect
+
+The Client SDK has functions for explicitly reconnecting a call or conversation's media. This is useful for example when you want to switch which device a user is speaking on without hanging up the call and starting a new one if the application dies.
```
Comment (conshus) [inline _documentation/en/client-sdk/in-app-voice/guides/reconnect-call.md:33]:
```suggestion
The Client SDK has functions for explicitly reconnecting a call or conversation's media. This is useful, for example, when you want to switch which device a user is speaking on without hanging up and starting a new call if the application dies.
```

**Pull request merged**